core/services/wifi/WifiManager.py

```diff
@@ -108,21 +108,14 @@ async def set_wifi_password(self, credentials: WifiCredentials) -> Any:
         """
         try:
             data = await self.wpa.send_command_add_network()
-            data = data.strip()
-            if data == b"Fail":
```

Comment:
I think you are mixing them. This one was already there, but I've removed since we now check for FAIL by default in every command.

Reply:
But capitalized FAIL is different from Fail, that is also different from FAIL-BUSY, capitalization matter on the string comparison. 
Where is the current patch checking all possible fails strings ?

Reply:
I'm pretty sure wpa_cli uses capitalized FAIL for all cases. Did you ever experienced a non-capitalized one?
Anyway, if you're afraid this can happen, we can just capitalize the received answer on the `send_command` check and compare just one possibility, covering all.

Reply:
Just checked [here](https://github.com/cozybit/hostap-sae/blob/master/wpa_supplicant/wpa_cli.c) and it's always capitalized, indeed.
This check on the `add_network` was in fact never working.